xclim/indices/_agro.py

```diff
@@ -886,12 +886,11 @@ def water_budget(
 
 @declare_units(
     pr="[precipitation]",
-    pr_cal="[precipitation]",
 )
 def standardized_precipitation_index(
     pr: xarray.DataArray,
-    pr_cal: Quantified,
-    freq: str = "MS",
```

Comment:
We could add `params` as a possible input if keeping the possibility of user-inputted calibration is still wanted